### DISCUSSION TITLE
import asynctelnet

### DISCUSSION THREAD

**Discussion by dy-nz** [custom-components/pyscript#481] [Q&A] [answered]
Hi. I'm running:
Home Assistant 2023.7.0
Supervisor 2023.07.1
Operating System 10.3
Frontend 20230705.0 - latest
Also, 
   pyscript:
     allow_all_imports: true
     hass_is_global: true

I'd like to move from sync telnet to async telnet.

I can "import anyio" fine.
In this very locked down HA environment, how do I "import asynctelnet" ?

Do I need to download the source and reassemble it in HA by hand ?
Thanks.

**Comment by dy-nz** [comment]
My effective workaround is to "wrap" my telnet function with:
@pyscript_executor

**Comment by craigbarratt** [comment, marked as answer]
Ah - that's a good solution!
